### PR TITLE
Upgrade to Webpack 2 and Elm 0.18

### PR DESCRIPTION
**Pull request by jiwhiz**
Make the example running with Webpack 2.2 and Elm 0.18.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Elm CSS loader [![Travis build Status](https://travis-ci.org/tcoopman/elm-css-webpack-loader.svg?branch=master)](http://travis-ci.org/tcoopman/elm-css-webpack-loader)
+# Elm CSS loader [![Travis build Status](https://travis-ci.org/jiwhiz/elm-css-webpack-loader.svg?branch=upgrade)](http://travis-ci.org/jiwhiz/elm-css-webpack-loader)
```
Comment (ahstro) [inline README.md]: Should probably not be in the commit, right?

Comment (jiwhiz) [reply README.md]: fixed

**Comment by ahstro** [comment]
Awesome, just in time for my migration at work :)

**Comment by ahstro** [comment]
Mind publishing this on npm soon? 😇

**Comment by tcoopman** [comment]
I'm not on a pc where I can publish this at the moment. I'll do it later
today

**Comment by ahstro** [comment]
Thanks :)

**Comment by tcoopman** [comment]
published as 3.0.0

**Pull request merged**